src/js/jsx/sections/libraries/LibrariesPanel.jsx

```diff
@@ -100,72 +100,68 @@ define(function (require, exports, module) {
             this.getFlux().actions.libraries.removeCurrentLibrary();
         },
 
-        render: function () {
+        _containerContents: function () {
             var libraryStore = this.getFlux().store("library"),
-                connected = libraryStore.getConnectionStatus(),
-                libraries = this.state.libraries,
-                currentLibrary = libraries.get(this.state.selectedLibrary);
-
-            var containerClasses = classnames({
-                "section-container": true,
-                "section-container__collapsed": !this.props.visible
-            });
-
-            var sectionClasses = classnames({
-                "libraries": true,
-                "section": true,
-                "section__sibling-collapsed": !this.props.visibleSibling
-            });
+                connected = libraryStore.getConnectionStatus();
 
             var containerContents;
```

Comment:
Let's consolidate these lines so containerContents is not in a new var line and is grouped with others above.